syntax = "proto3";

message Nada{
}

message Mensaje{
	string timestamp = 2;
	string contenido = 3;
}

service ServerChat {
	rpc ChatStream (Nada) returns (stream Mensaje);
	rpc SendMessage (Mensaje) returns (Nada);
}